easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+#
+# Fred Hutchenson Cancer Research Center Seattle WA
+name = 'OpenJPEG'
+version = '2.3.0'
+
+easyblock = 'CMakeMake'
```

Comment:
@fizwit For consistency with existing easyconfig files, this should go above the `name =` line (with an empty line in between).

Reply:
extreme badness - forgetting the sanity_check_paths